packages/medusa/src/strategies/batch-jobs/product/import.ts

```diff
@@ -600,9 +655,30 @@ class ProductImportStrategy extends AbstractBatchJobStrategy {
         delete variant.id
         delete variant.product
 
-        await this.productVariantService_
-          .withTransaction(transactionManager)
-          .create(product!, variant as unknown as CreateProductVariantInput)
+        if (isMedusaV2Enabled) {
```

Comment:
todo: In general, the workflows accept a collection of element, we should use that batch approach in a follow up pr. cc @olivermrbl @carlos-r-l-rodrigues